app/src/main/java/cm/aptoide/pt/home/BottomNavigationActivity.java

```diff
@@ -31,22 +21,24 @@
     implements AptoideBottomNavigator {
 
   protected static final int LAYOUT = R.layout.frame_layout;
-  private final static String EVENT_ACTION =
-      "https://ws75.aptoide.com/api/7/getStoreWidgets/store_id=15/context=stores";
+  private final String ITENS_LIST_KEY = "BN_ITENS";
   protected BottomNavigationView bottomNavigationView;
-  @Inject BottomNavigationAnalytics bottomNavigationAnalytics;
-  @Inject SearchAnalytics searchAnalytics;
+  @Inject BottomNavigationMapper bottomNavigationMapper;
+  @Inject BottomNavigationNavigator bottomNavigationNavigator;
   private PublishSubject<Integer> navigationSubject;
   private Animation animationup;
   private Animation animationdown;
 
   @Override protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
-    getActivityComponent().inject(this);
-
     setContentView(LAYOUT);
     navigationSubject = PublishSubject.create();
     bottomNavigationView = (BottomNavigationView) findViewById(R.id.bottom_navigation);
+    super.onCreate(savedInstanceState);
```

Comment:
if a fragment needs to do something with the activity onCreate it will break if it has it below. I can change it back, because right now it has no use having it as it is